viewer-prototype/src/browser/trace-viewer/trace-viewer-frontend-module.ts

```diff
@@ -4,7 +4,8 @@ import { TraceViewerWidget, TraceViewerWidgetOptions } from './trace-viewer';
 import { TraceViewerContribution } from './trace-viewer-contribution';
 import { TraceViewerEnvironment } from '../../common/trace-viewer-environment';
 import { TraceServerUrlProvider } from '../../common/trace-server-url-provider';
-import { CommandContribution } from '@theia/core/lib/common';
+import { CommandContribution, MenuContribution } from '@theia/core/lib/common';
```

Comment:
I wonder if we should add the command to the main File menu (see https://github.com/theia-ide/theia-trace-extension/issues/159#issuecomment-733749083). So, I think the feature to add it to the main menu should be a different PR. That would allow us to merge the feature of "Open Trace" button (#160) first and decide the way forward for #159.